packages/patternfly-4/react-core/src/components/SimpleList/SimpleListGroup.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/SimpleList/simple-list';
+
+export interface SimpleListGroupProps {
```

Comment:
should extend HTMLProps for section

Reply:
I could only find `React.HTMLProps<HTMLTableSectionElement>`, would this be correct? It seems to work with the console logging warnings

Reply:
hm, how about passing `<any>` in this case since we just want the default HTML attributes